handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -1086,6 +1086,25 @@ export default () => {
      * @example
      * ```js
      * // freeze the first 3 columns from the left
+     * fixedColumnsStart: 3,
+     * ```
+     */
+    fixedColumnsStart: 0,
+
+    /**
+     * An alias for `fixedColumnsStart`. Works only for Left-To-Right.
+     * Throws a warn when used.
+     * Throws an error when used together with `fixedColumnsStart`.
+     * Throws an error when used in RTL mode.
```

Comment:
```suggestion
     * The `fixedColumnsLeft` option is an alias for the [`fixedColumnsStart`](#fixedColumnsStart) option.
     *
     * The `fixedColumnsLeft` option works only in the left-to-right mode, and:
     * - Throws a warning when used
     * - Throws an error when used together with the [`fixedColumnsStart`](#fixedColumnsStart) option
     * - Throws an error when used in the right-to-left mode
```

Reply:
`* - Throws a warning when used` is in-actual for current assumes.
```suggestion
     * The `fixedColumnsLeft` option is an alias for the [`fixedColumnsStart`](#fixedColumnsStart) option.
     *
     * The `fixedColumnsLeft` option works only in the left-to-right mode, and:
     * - Throws an error when used together with the [`fixedColumnsStart`](#fixedColumnsStart) option
     * - Throws an error when used in the right-to-left mode
```